documentapi/src/main/java/com/yahoo/documentapi/messagebus/protocol/DocumentProtocol.java

```diff
@@ -66,7 +66,6 @@
     public static final int MESSAGE_EMPTYBUCKETS = DOCUMENT_MESSAGE + 23;
     public static final int MESSAGE_REMOVELOCATION = DOCUMENT_MESSAGE + 24;
     public static final int MESSAGE_QUERYRESULT = DOCUMENT_MESSAGE + 25;
-    public static final int MESSAGE_BATCHDOCUMENTUPDATE = DOCUMENT_MESSAGE + 26;
```

Comment:
This package is marked as PublicApi in package-info.java, so public fields cannot be removed from the master branch, unless you make sure nobody is using them.